src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -62,6 +64,16 @@ public void getTagList_modifyList_throwsUnsupportedOperationException() {
         addressBook.getTagList().remove(0);
     }
 
+    @Test
+    public void getTodayReviewList() {
+        AddressBook newData = getTypicalAddressBook();
+        addressBook.resetData(newData);
+        List<Card> expectedList = addressBook.getCardList();
+        List<Card> actualList = addressBook.getTodayReviewList();
+        assertEquals(expectedList, actualList);
```

Comment:
why are they supposed to be equal?

Reply:
all new posts added are for review immediately. Predicate should return everything.
Easy test to do for now.

Reply:
this is not equal all the time, so the test is invalid.